frontend/package.json

```diff
@@ -96,15 +96,20 @@
     "instituteContacts": "node tests/e2e/testSuites/instituteE2E/instituteContactsTest.js",
     "instituteDetails": "node tests/e2e/testSuites/instituteE2E/instituteDetailsTest.js",
     "createNewMessageWNote": "node tests/e2e/testSuites/edxE2E/edxMessageTest.js",
-    "viewSchoolContacts": "node tests/e2e/testSuites/instituteE2E/viewSchoolContactsTest.js"
+    "viewSchoolContacts": "node tests/e2e/testSuites/instituteE2E/viewSchoolContactsTest.js",
+    "insertSchoolPrincipal": "node tests/e2e/testCases/studentAdmin/institute/insertSchoolPrincipal.js",
+    "deleteSchoolPrincipal": "node tests/e2e/testCases/studentAdmin/institute/deleteSchoolPrincipal.js"
   },
   "dependencies": {
     "axios": "^0.27.2",
     "date-and-time": "^0.14.2",
     "dotenv": "^8.2.0",
     "nconf": "^0.12.0",
     "npmlog": "^4.1.2",
-    "testcafe": "^2.0.1"
+    "testcafe": "^2.0.1",
+    "@js-joda/core": "^2.0.0",
```

Comment:
I don't think we need js-joda/core after we remove the schoolUtil functions that aren't used.